client/src/pages/Profile.tsx

```
@@ -197,265 +227,345 @@ const Dashboard = () => {
+														dangerouslySetInnerHTML={{ __html: streakStat }}
+														className="overflow-x-auto"
+													/>
+											) : (
+												<></>
+											)}
+										</div>
+										<div>
+											<h3 className="font-semibold mb-2">GitHub Streak</h3>
+											{githubStreakSvg ? (
+													<div
+														dangerouslySetInnerHTML={{ __html: githubStreakSvg }}
```

Biome

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)